app/presenters/edition_presenter_helper.rb

```diff
@@ -8,7 +8,9 @@ def as_hash
       url: url,
       organisations: display_organisations.html_safe,
       display_date_microformat: display_date_microformat,
-      public_timestamp: model.public_timestamp
+      public_timestamp: model.public_timestamp,
+      historic?: model.historic?,
+      government_name: government_name
```

Comment:
Any reason why this doesn't call `model.search_government_name` directly the same as `model.historic?` and goes via an extracted method?